daemon/checkpoint.go

```diff
@@ -17,6 +18,13 @@ var (
 	validCheckpointNamePattern = names.RestrictedNamePattern
 )
 
+// Checkpoint represents the details of a checkpoint
+type Checkpoint struct {
```

Comment:
If this type is (at least currently) only used locally, perhaps it should be un-exported.

Reply:
Thank you.
I should be careful every time. Don't just copy & paste.